static/examples/cstg-secure-signals-example/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>UID2 Publisher Client-Side Integration Example using UID2 JavaScript SDK, Secure Signals</title>
+    <link rel="stylesheet" type="text/css" href="./styles/ads.css" />
+    <link rel="stylesheet" type="text/css" href="./styles/app.css" />
+    <link rel="shortcut icon" href="images/favicon.png" />
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script>
+
+    </script>
+  </head>
+  <body>
+    <h1>UID2 Publisher Client-Side Integration Example using UID2 JavaScript SDK, Secure Signals</h1>
+    <p>
+      This example demonstrates how a content publisher can follow the
+      <a href="https://unifiedid.com/docs/guides/integration-javascript-client-side"
```

Comment:
Ah oops I missed this too. The link is to the UID2 doc, but this is EUID. The URL is:
https://euid.eu/docs/guides/integration-javascript-client-side
Doc title is the same.